build/build.py

```diff
@@ -432,8 +437,7 @@ def main():
   os.chdir(os.path.dirname(__file__ or args.prog) or '.')
 
   # Find a working Bazel.
-  bazel_path = get_bazel_path(args.bazel_path)
-  check_bazel_version(bazel_path, min_version="2.0.0", max_version=None)
+  bazel_path = get_bazel_path(args.bazel_path, min_version="2.0.0", max_version=None)
```

Comment:
My understanding is that the min_version is 2.0.0 here because that the min version that compares against .bazelversion. If so, maybe remove the min_version and max_version arguments here and just check directly in `check_bazel_version` that the version is >= 2?

Reply:
Yes, this is exactly what I wanted to do but I wasn't sure whether it was appropriate with you.

To be honest I would like to remove this check completely and update installation guide with requirements to bazel version to be >=2. This version check will become obsolete sometime and will be removed. Maybe it is out of date? I known nothing about bazel 2.0.0 adoption rate but it was released 19 December 2019.

Reply:
I also have no idea what bazel versions most people are running. I think we should keep the 2.0.0 check in this file just in case -- if someone is using an ancient bazel version, it'll really help with debugging, since using an old bazel usually produces a confusing error message. I can also imagine people having really old bazel versions laying around since relatively few projects use bazel.

I'm fine with this as-is, but WDYT of at least removing the min_version/max_version arguments here and hardcoding the 2.0.0 version check? I think that'll make this file less confusing.